main.py

```diff
@@ -458,9 +458,9 @@ def parse_leaders(metric: Metric, data: Tag) -> t.List[MetricLeader]:
     return [parse_leader(metric, row) for row in rows]
 
 
-def build_url(metric: Metric) -> str:
+def build_url(metric: Metric, page: int) -> str:
```

Comment:
We need to also pass this `page` parameter on line 479.

```py
url = build_url(metric, 1)
```

Reply:
lol you fixed it before i could even review.

Now that's good customer service.